MIGRATE.md

```diff
@@ -52,16 +52,17 @@ server or note its path if the same server will be used.
 
 1. Make sure your rmt installation is up-to-date. `rmt-data-import` is available in RMT versions >= 1.0.0
 2. Unpack the tarball containing SMT data to some directory, e.g. `/root/smt-data`
-3. If you chose to export SMT's SSL certificates, decrypt the SMT CA private key to `/usr/share/rmt/ssl/`:
+3. If you chose to export SMT's SSL certificates, copy the SMT CA private key to `/usr/share/rmt/ssl/`:
     ```
-    openssl rsa -in /root/smt-data/ssl/cacert.key -out /usr/share/rmt/ssl/rmt-ca.key
+    cp /root/smt-data/ssl/cacert.key /usr/share/rmt/ssl/rmt-ca.key
     ```
+    It is strongly recommended to encrypt your CA private key.
 4. If you chose to export SMT's SSL certificates, copy the SMT CA certificate to `/usr/share/rmt/ssl/`:
     ```
     cp /root/smt-data/ssl/cacert.pem /usr/share/rmt/ssl/rmt-ca.crt
     ```
 5. Run YaST RMT configuration module from YaST command center or by running `yast2 rmt` on the command line;
-6. Go through setup steps of the YaST module. On the SSL setup page it is possible to add alternative common names (e.g., the hostname of the SMT server in case it is desirable to perform a switch) 
+6. Go through setup steps of the YaST module. On the SSL setup page it is possible to add alternative common names (e.g., the hostname of the SMT server in case it is desirable to perform a switch)
```

Comment:
Maybe we can make this more clear: 

Proceed through the YaST module. If you want to support the old SMT hostname, you can add it as an *alternative common name* on the SSL setup page.

Reply:
Let's summon @svenseeberg for that maybe :grimacing:

Reply:
Sure, I wasn't so happy with "in case it is desirable to perform a switch". The language was a bit too ambiguous for me.